scripts/align-version.sh

```diff
@@ -14,7 +14,7 @@ ${scriptdir}/align-version.js ${files}
 
 # validation
 marker=$(node -p "require('./scripts/get-version-marker')")
-if find . -name package.json | grep -v node_modules | xargs grep "[^0-9]${marker}"; then
+if find . -name package.json | grep -v node_modules | xargs grep -P "\"[^0-9]?${marker}\""; then
```

Comment:
I wanted to suggest something like that on initial review of that regex.

It should however catch *all* kinds of quotes (backticks, single, double - minimum).

Reply:
Alternatively, we could ignore quotes and exclude an additional `.0` that'd make it into an IP address.

Reply:
in fact when switching to regex here (`-P`) we have a problem with the dots (`.`) coming from the `0.0.0` (exported from js) not being escaped

Reply:
and it should be `-E`

Reply:
@RomainMuller why would you want to check for all kinds of quotes, it's a find in package.json files and other quotes are not valid json, no?